### ISSUE TITLE
Gardener landscape not working after 0.6 release

### ISSUE THREAD

**Issue by donCappo** [danielfoehrKn/kubeswitch#33]
Hi Daniel,

I'm struggling to make switcher load Gardener landscape(s) after 0.6 release. I'm aware of the new AdminKubeconfigRequest, which I have enabled at apiserver. I also tested the functionality of AdminKubeconfigRequest via creating the request manually and also via gardenlogin kubeconfig which also works.
When I downgrade back to 0.5 then gardener landscape load correctly.
Only extra output I get, when running switch with debug.

`switcher --debug --no-index --show-preview=false`
`DEBU[0000] Using kubeconfig path from flag /Users/..../.kube/config`
`DEBU[0000] Added store with kind gardener and ID gardener.garden-.....`
`DEBU[0000] Added store with kind filesystem and ID filesystem.env-and-flag`

which seem to me be that gardener being added correctly

any hint would be great

many thanks
Pavel

**Comment by danielfoehrKn** [comment]
Hi,
I'll have a look. Unfortunately, the debug logs are not helpful.

Interesting, I use the Gardener store daily in our SAP landscapes.
Have you tried using [gardenctlv2](https://github.com/gardener/gardenctl-v2) and did that work?
I am asking because they use the same `AdminKubeconfigRequest` underneath.

**Comment by donCappo** [comment]
Hi Daniel, sorry for getting back so late. I think I figured it out, or more like your pointing to gardenctl-v2 have helped.
So apparently gctl-v2 requires ... still (https://github.com/gardener/gardenctl-v2/pull/149) to have existing kubeconfigs in virtual garden since i was getting errors like
`Error: failed to get kubeconfig for shoot garden-devel/dev: failed to get configmap garden-devel/dev.kubeconfig: configmaps "dev.kubeconfig" not found`
that lead me to the [gardenlogin-controller-manager](https://github.com/gardener/gardenlogin-controller-manager) which we don't have deployed.
So just to test it I've created CM kubeconfig manually and gctl-v2 worked and also switcher showed the target cluster. So partially it's my fault, but on the other hand it is not much documented :-)
